### DISCUSSION TITLE
Quotes widget - Random not working?

### DISCUSSION THREAD

**Discussion by ryanbuening** [bluecaret/carettab#344] [Help and Support] [answered]
Adding multiple quotes and enabling the "Random" feature does not appear to be working for me.

**Comment by bluecaret** [comment, marked as answer]
Thank you for letting me know, I'll look into it. But just to be clear, can you check back in a day? The quotes are not random every new tab, but instead update every day, whether set to random or not.

**Reply by ryanbuening** [reply]
Ah, interesting. Is that a change from 3.x? I thought on 3.x, the quote would be random on each new tab.

**Reply by bluecaret** [reply]
Possibly. I'll be honest and don't really remember at this point. :) Would you prefer an every tab update? I could potentially make that an option.

**Reply by ryanbuening** [reply]
Got it. I think that would be a nice option if it isn't much work. :)

4.x is looking great btw!

**Reply by ryanbuening** [reply]
Just wanted to report that the quote _does_ change after a day.

**Reply by bluecaret** [reply]
Good to know, thanks!
